src/tip-opengov.ts

```diff
@@ -122,7 +122,7 @@ async function signAndSendCallback(
       bot.log(msg, result.status);
       reject({ success: false, errorMessage: msg });
     } else {
-      bot.log(`Tip for ${contributor.address} ${type} status: ${result.status.type}`, result.status);
+      bot.log(`Tip for ${contributor.address} ${type} status: ${result.status.type}`);
```

Comment:
Judging by [the types](https://github.com/polkadot-js/api/blob/master/packages/types/src/interfaces/author/types.ts#L34), you were probably looking for `result.status.type` here